grammar.js

```diff
@@ -248,6 +253,7 @@ module.exports = grammar({
             alias($.inline_comment_close, "_lowercase"),
             alias($.inline_math_close, "_lowercase"),
             alias($.variable_close, "_lowercase"),
+            alias($.inline_link_target_close, "_lowercase"),
```

Comment:
The question is here: should we also add link description or text `_end` nodes and make them lowercase if they do not have a matching beginning?